Package cannot install on node > v14 with npm but works for yarn

**Describe the bug**
With node 16.x you get errors on install.

**To Reproduce**
1. `git clone repo`
2. `npm install`
3.

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @cubejs-client/core@0.30.46
npm ERR! Found: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^7.21.0" from @cubejs-client/core@0.30.46
npm ERR!   packages/cubejs-client-core
npm ERR!     @cubejs-client/core@0.30.46
npm ERR!     node_modules/@cubejs-client/core
npm ERR!       workspace packages/cubejs-client-core from the root project
npm ERR!       7 more (@cubejs-backend/testing, @cubejs-client/ngx, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@"^4.19.1 || ^5.3.0" from eslint-config-airbnb-base@13.2.0
npm ERR! node_modules/eslint-config-airbnb-base
npm ERR!   dev eslint-config-airbnb-base@"^13.1.0" from @cubejs-client/core@0.30.46
npm ERR!   packages/cubejs-client-core
npm ERR!     @cubejs-client/core@0.30.46
npm ERR!     node_modules/@cubejs-client/core
npm ERR!       workspace packages/cubejs-client-core from the root project
npm ERR!       7 more (@cubejs-backend/testing, @cubejs-client/ngx, ...)
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/josx/.npm/eresolve-report.txt for a full report.
```

**Expected behavior**
Clear installation without errors.

**Version:**
master branch

**Fix nvm use 14*

--- a/package.json
+++ b/package.json
@@ -64,11 +64,7 @@
     "prettier": "^2.0.5",
     "rimraf": "^3.0.2",
     "rollup": "2.53.1",
-    "rollup-plugin-less": "^1.1.3",
-    "rollup-plugin-local-resolve": "^1.0.7",
     "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-postcss": "^4.0.0",
-    "rollup-plugin-uglify": "^6.0.4",
     "typescript": "~4.1.5"
   },
   "repository": {

